### PR TITLE
[7] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by Lolatech07**
## Task: `Develop a Contact Form Page using HTML`
- `submission of task
Screenshot of Contact Form Page

`

### CODE REVIEW

File: submissions/lolaOluwadahunsi/html/ecommerce-website.html [hunk 1]
```diff
@@ -34,8 +34,13 @@ <h2><strong>ABOUT LAPTOPS R US</strong></h2>
         </p>
     </article>
 
+<<<<<<< HEAD:submissions/lolaOluwadahunsi/html/Ecommerce website.html
```
Comment (adaugochi) [inline submissions/lolaOluwadahunsi/html/ecommerce-website.html:37]: Resolve this merge conflict you have from line 37 - 43.

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by Lolatech07** [comment]
Hi. Sorry but I don't understand this instruction.

If you please help clarify. Thanks.

File: submissions/lolaOluwadahunsi/html/ecommerce-website.html (hunk 1 above)
Comment (ToluAkin) [inline submissions/lolaOluwadahunsi/html/ecommerce-website.html:37]: Delete this line @Lolatech07

File: submissions/lolaOluwadahunsi/html/ecommerce-website.html [hunk 2]
```diff
@@ -34,8 +34,13 @@ <h2><strong>ABOUT LAPTOPS R US</strong></h2>
...
+    <img src="images/header%20image%201.jpg" width="1200" height="600" alt="Main Header" content>
+    <img src="images/header%20image%202.jpg" width="1200" height="600" alt="Header" content>
+=======
```
Comment (ToluAkin) [inline submissions/lolaOluwadahunsi/html/ecommerce-website.html:40]: Delete this line

File: submissions/lolaOluwadahunsi/html/ecommerce-website.html [hunk 3]
```diff
@@ -34,8 +34,13 @@ <h2><strong>ABOUT LAPTOPS R US</strong></h2>
...
     <img src="images/header_image_1.jpg" content>
     <img src="images/header_image_2.jpg" content>
+>>>>>>> 46c93c76184ec4d37096ee47fca906118fe7205d:submissions/lolaOluwadahunsi/html/ecommerce-website.html
```
Comment (ToluAkin) [inline submissions/lolaOluwadahunsi/html/ecommerce-website.html:43]: Delete this line

File: submissions/lolaOluwadahunsi/html/ecommerce-website.html [hunk 4]
```diff
@@ -34,8 +34,13 @@ <h2><strong>ABOUT LAPTOPS R US</strong></h2>
         </p>
     </article>
 
+<<<<<<< HEAD:submissions/lolaOluwadahunsi/html/Ecommerce website.html
+    <img src="images/header%20image%201.jpg" width="1200" height="600" alt="Main Header" content>
```
Comment (ToluAkin) [inline submissions/lolaOluwadahunsi/html/ecommerce-website.html:38]: You need to remove the attributes `width` and `height`. Also, update the name of your images that is not how they were saved.

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Comment by Lolatech07** [comment]
How do I delete please?

**Comment by Lolatech07** [comment]
I think I sort of figured it out...kindly review. Thanks.

File: submissions/lolaOluwadahunsi/htmlForms/contact_form_page.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Contact Form</title>
+</head>
+<body>
+<h1>Contact Form</h1>
+<form action="#" method="POST" enctype="application/x-www-form-urlencoded">
```
Comment (ToluAkin) [inline submissions/lolaOluwadahunsi/htmlForms/contact_form_page.html:9]: @Lolatech07 Remove attributes `action`, `method` and `enctype`. They are not needed for now.

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Comment by Lolatech07** [comment]
Thank you. Attributes have been removed.

**Review by ToluAkin — APPROVED** [review]

**Pull request merged**